### ISSUE TITLE
Clarification of compatibility with iban-js

### ISSUE THREAD

**Issue by waldyrious** [koblas/ibankit-js#2]
The README claims that ibankit is "Drop-in replaceable with iban-js", but it's unclear what that means. Would a project using ibankit be able to swap to iban-js and continue working, or is it the opposite? The latter seems more likely (and fits better with what the README mentions further down about "porting from iban-js applications", but the language suggests the contrary.

Perhaps "Drop-in replacement for iban-js" would better match the intention?

**Comment by waldyrious** [comment]
Ok, cool, I will submit that change as a PR. But before that there's one further detail that needs clarification: from [`iban-js`'s API](https://www.npmjs.com/package/iban#api), only `isValid()` seems to be implemented by `ibankit`. None of `toBBAN()`, `fromBBAN()`, `isValidBBAN()`, `printFormat()` and `electronicFormat()` seem to be supported. So perhaps `ibankit` is not a full replacement for `iban-js`, but just to its `isValid` method, right?

Is that intentional, or would adding support for those functions be within the roadmap for `ibankit`? I personally think that would be nice since it would allow downstream users to seamlessly transition to `ibankit`, which IMO offers a nicer and more complete API.

**Comment by koblas** [comment]
All of those functions would be pretty simple to add since the functionality fundamentally exists. 

Is it functionality that you think people are really using since the README really focuses on isValid().

**Comment by waldyrious** [comment]
I can't speak for others, but I have worked in code that indeed uses, for instance, `electronicFormat()` (I found that out precisely by attempting a drop-in replacement by changing the `require` line).

So I definitely think there's value in offering 100% feature-parity with iban-js. It's a small API anyway, so for little effort `ibankit` could guarantee that transitioning from `iban-js` is totally painless. That's a pretty great trade-off, I think. :)
